### PR TITLE
View

### PR DESCRIPTION
**Pull request by hakichris**
In this branch, we are creating UI for the app. 

- [x] no linter error
- [x] GitFlow 
- [x] view works properly

### CODE REVIEW

File: app/views/users/_userid.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+   </ul>
+</div>
+<div class="post">
+    <ul class="post_info width"> 
+    <h3 class="title"><%= @post.title%> #<%= @post.id%></h3>
+   <li class="bio_content"> <%= @post.text %> </li>
+   <li class="list_countcomentlikes">Comments: <%= @post.commentscounter %>, Likes: <%= @post.likescounter %> </li>
+   </ul>
+</div>
+<button class="allcoment"> See all posts</button>
```
Comment (CollinsTatang) [inline app/views/users/_userid.html.erb]: - [ ] Please add the link to display all user post in your `user/user_show.html.erb` file as shown on the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAACB75ZFPD6AMW3EAVIL24SGMZDGAQNA)
```ruby
<button>
<%= link_to 'See all Post', user_posts_path(@user) %>
</button>
```

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @hakichris , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/_userid.html.erb (hunk 1 above)
Comment (hakichris) [reply app/views/users/_userid.html.erb]: done, thank you.

Comment (hakichris) [reply app/views/users/_userid.html.erb]: done

**Review by gemmen29 — APPROVED** [review]
Hi @hakichris, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**